packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -4,23 +4,82 @@ import { error_types, unrecoverable_errors, observer, message_types } from '@der
 import { contract_stages } from '../constants/contract-stage';
 import { switch_account_notification } from '../utils/bot-notifications';
 
+const statistics_storage_key = 'statistics_cache';
+
+const empty_statistic = {
+    lost_contracts: 0,
+    number_of_runs: 0,
+    total_profit: 0,
+    total_payout: 0,
+    total_stake: 0,
+    won_contracts: 0,
+};
+let get_statistics;
+
 export default class RunPanelStore {
     constructor(root_store) {
         this.root_store = root_store;
         this.dbot = this.root_store.dbot;
         this.registerCoreReactions();
+
+        get_statistics = JSON.parse(sessionStorage.getItem(statistics_storage_key));
```

Comment:
Since we're calling `JSON.parse(sessionStorage.getItem(statistics_storage_key))` multiple times throughout this file, could we create a function out of it? This function would handle a situation where there aren't any sessionStorage values too.